### ISSUE TITLE
Fix broken links

### ISSUE THREAD

**Issue by WrinklyNinja** [loot/skyrim#89]
pStyl3 reported a few broken links, a couple of which I haven't yet fixed:

```
http://www.nexusmods.com/skyrim/mods/18110
http://www.nexusmods.com/skyrim/mods/17993
```

These are for hair mods, and there are a few similar mods that may be compatible forks, but they need more investigation.

**Comment by LuthienKess** [comment]
I took a look into these two links. Both were originally from the author [radioragae](http://www.nexusmods.com/skyrim/users/3688265), but she/he removed them from the nexus in [late 2013](http://www.loverslab.com/topic/19129-where-can-i-find-non-adult-skyrim-requests/page-53#entry566462). 
The first mod was Hair Packs of TES IV Oblivion; I believe the file was "oblivion hair.esm".
The second was Hair Packs of Fallout 3; I'm not sure what the file was called.
Only 4 entries in the masterlist contain the first link, and 2 to the second. (No More Vanilla Hair contains both.)

* OblHairs(\d\d)?\.esm

I assume this was the original entry for Oblivion Hair (which would explain why I can't find it on the nexus). I'm not familiar with the formatting, but if it was the entry for the original mod, we could remove it. (All the entry contains is a link to the mod page and a reminder to follow the instructions therein.)

* Taleya.esp

This mod has been merged into [Vizerin and Company](http://www.nexusmods.com/skyrim/mods/21749). According to the description Oblivion Hair is no longer needed, as the file now depends on ApachiiSkyHair instead. I think that since this esp is no longer available (even in the old versions section of the nexus page) we can remove the entry completely.

* VizerinChronicles.esp

I believe this was a mod that Taleya.esp was merged with to create Vizerin and Company. I wasn't able to find the file on the nexus. Again, I think we could remove this entry from the masterlist.


* Onmeya.esp

Is another mod that is no longer on the nexus. The author says [here](http://www.nexusmods.com/skyrim/mods/21540) that Onmeya has been removed for updating. I think we could remove this entry since when the author re-uploads the mod, it will almost certainly not depend on a mod that isn't available.

* NoMoreVanillaHair.esp

This one's a bit trickier. The [mod](http://www.nexusmods.com/skyrim/mods/19814) hasn't been updated since 2012, and still lists both Oblivion Hair and Fallout 3 Hair as requirements. I tested in-game without those two files, and didn't run into any crashes or glitches; it just seems that the mod didn't do what it was meant to, namely replace vanilla hair. I'm not sure how we want to handle this.

Just to be complete, I was able to find a newer [mod](http://www.nexusmods.com/skyrim/mods/22598) that is an update of the original Oblivion Hair mod. Presumably mod authors could change their dependencies to this mod if they needed.


I could remove the first 4 entries, but I'm not sure what to do about the last one. Suggestions?

**Comment by WrinklyNinja** [comment]
Yep, removing those that aren't available any more seems OK, since their entries don't contain any other useful metadata that would otherwise make keeping them a good idea.

`NoMoreVanillaHair.esp` sounds like it used the resources from Oblivion Hair, if it could run without crashing without the latter. Perhaps add a warning saying that it depends on mods that are no longer available? 

The updated version of Oblivion Hair may work with it, but the Fallout 3 hair pack doesn't exist, and I suspect that Oblivion Hair falls foul of Bethesda's policy of not allowing cross-game mods anyway (unless it's adding hairs from Oblivion mods, in which case there's probably still a copyright issue). That's probably why the originals got removed, and the update probably only exists due to lack of exposure and so reporting.

**Comment by Ortham** [comment]
This got closed by c79f57ae1d4c61b100041eb93a22b082862bcd0f.
